### PR TITLE
[DOCS] Reformat cat alias API 

### PR DESCRIPTION
**Pull request by jrodewig**
Relates to elastic/docs#937

This PR updates the cat alias API to align with the new [API reference template](https://github.com/elastic/docs/blob/master/shared/api-ref-ex.asciidoc).

Dependent on #45117, which sets up the `common-parms.asciidoc` file.

### Preview
http://elasticsearch_45119.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/cat-alias.html

### CODE REVIEW

**Comment by jrodewig** [comment]
test this please

File: docs/reference/cat/alias.asciidoc [hunk 1]
```diff
@@ -1,8 +1,48 @@
...
+[[cat-alias-api-path-params]]
+==== {api-path-parms-title}
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=name]
+
+
+[[cat-alias-api-query-params]]
+==== {api-query-parms-title}
+
+// Format
```
Comment (nik9000) [inline docs/reference/cat/alias.asciidoc]: I'm not sure the comments help all that much given the fairly descriptive tag names.

Comment (jrodewig) [reply docs/reference/cat/alias.asciidoc]: Thanks @nik9000. Removed with 3e43b94.

**Review by nik9000 — APPROVED** [review]
LGTM

**Review by szabosteve — APPROVED** [review]
LGTM

**Pull request merged**